html/components/_footer.scss

```diff
@@ -0,0 +1,41 @@
+// ==================================================================
+// Footer
+// ==================================================================
+
+////
+/// @group Footer
+////
+.sprk-c-Footer {
+  background-color: $sprk-footer-background-color;
+}
+
+.sprk-c-Footer__text {
+  color: $sprk-footer-text-color;
+}
+
+.sprk-c-Footer__link {
+  color: $sprk-footer-link-color;
+  font-weight: $sprk-footer-link-font-weight;
+}
+
+.sprk-c-Footer__icon {
+  fill: $sprk-footer-icon-color;
+  color: $sprk-footer-icon-color;
+}
+
+.sprk-c-Footer .sprk-c-Footer__trigger {
```

Comment:
Add comment that this is the toggle and needs to be overridden because of the dark background of footer